graphql/schema.graphql

```diff
@@ -15,14 +15,25 @@ input DateRange {
 type User {
     name_first: String!
     name_last: String!
-    id: ID
+    id: ID!
     atcRating: Rating!
 }
 
 type Rating {
     code: String!
 }
 
+interface Node {
```

Comment:
Not quite sure how useful having this as an interface is?

Reply:
https://atheros.ai/blog/graphql-interfaces-and-unions-how-to-design-graphql-schema